jax/_src/scipy/stats/_core.py

```diff
@@ -147,3 +147,15 @@ def rankdata(
   if method == "average":
     return .5 * (count[dense] + count[dense - 1] + 1).astype(dtypes.canonicalize_dtype(jnp.float_))
   raise ValueError(f"unknown method '{method}'")
+
+@_wraps(scipy.stats.sem, lax_description="""\
+Currently the only supported nan_policy is 'propagate'
+""")
+@partial(jit, static_argnames=['axis', 'nan_policy', 'keepdims'])
+def sem(a: ArrayLike, axis: int | None = 0, ddof: int = 1, nan_policy: str = "propagate", *, keepdims: bool = False) -> Array:
+  check_arraylike("sem", a)
+  a = jnp.asarray(a)
```

Comment:
You may need to change the variable name here to prevent type checkers from complaining about use of array attributes below.